src/WebSocketServer.php

```diff
@@ -52,6 +52,10 @@ class WebSocketServer extends WssMain implements WebSocketServerContract
     private bool $stepRecursion = true;
 
     /**
+     * @var int
+     */
+    private int $loopingDelay = 1000;
```

Comment:
Does this delay time for 1ms solves the problem with CPU high consuming in practice? Was there any metrics? Is it 20%, 40% etc less f.e.?

Reply:
I'm using my own fork. With a 1ms delay the server CPU goes from 100% or more to about 6%. And using 2ms, the CPU is going about 2%.

Reply:
We can ask for milliseconds instead of microseconds of a second, and then set the usleep value x1000.

Reply:
> We can ask for milliseconds instead of microseconds of a second, and then set the usleep value x1000.

Sure it would be reasonable